.gitignore

```diff
@@ -16,3 +16,4 @@
 coverage.txt
 codegpt
 dist
+.idea/
```

Comment:
Unfortunately, the code patch you provided is incomplete and does not contain any code for review. Please provide the correct patch or the relevant code for review.